src/mmtk.rs

```diff
@@ -65,6 +66,12 @@ impl MMTKBuilder {
         self.options.set_bulk_from_command_line(options)
     }
 
+    /// Custom VM layout constants. VM bindings may use this function for compressed or 39-bit heap support.
+    /// This function must be called before MMTk::new()
+    pub fn set_custom_vm_layout(&mut self, constants: VMLayout) {
```

Comment:
Can we rename this to `set_vm_layout` to be consistent with the other function names? Or do others like `set_custom_vm_layout` more?

Reply:
Done